app/src/main/kotlin/org/fossify/messages/activities/SettingsActivity.kt

```diff
@@ -27,7 +27,7 @@ class SettingsActivity : SimpleActivity() {
     private var blockedNumbersAtPause = -1
     private var recycleBinMessages = 0
     private val messagesFileType = "application/json"
-    private val messageImportFileTypes = listOf("application/json", "application/xml", "text/xml")
+    private val messageImportFileTypes = listOf("application/json", "application/xml", "text/xml", "application/octet-stream")
```

Comment:
@tom93 why not do this at runtime for Android 8/9 only? That way, the behavior won't change on newer versions.